### PR TITLE
Update windows-sys to 0.45.0

### PR DESCRIPTION
**Pull request by messense**
https://github.com/microsoft/windows-rs/releases/tag/0.45.0

### CODE REVIEW

**Comment by messense** [comment]
Not sure what's going on with the failed tests.

File: src/schannel_cred.rs [hunk 1]
```diff
@@ -248,7 +246,7 @@ impl Builder {
 
             match Identity::AcquireCredentialsHandleA(
```
Comment (r-ml) [inline src/schannel_cred.rs:247]:
```suggestion
            match Identity::AcquireCredentialsHandleW(
```
`Identity::UNISP_NAME` is wide

**Comment by r-ml** [comment]
I get the remaining test failures on the master branch as well when testing locally :shrug:

**Comment by zh-jq** [comment]
now we have windows-sys 0.48

**Comment by djc** [comment]
@steffengy between #90 and #89, what's the plan here going forward? Would be nice to keep things moving.

**Comment by KapJI** [comment]
Any chances it can be upgraded to windows-sys 0.48 anytime soon?

**Comment by steffengy** [comment]
Dont see a reason not to merge an update to 0.48 that passes tests.

**Comment by messense** [comment]
Closing in favor of #93

**Pull request closed without merging**